README.md

```diff
@@ -17,18 +17,21 @@ curl https://vapor-ware.github.io/gimme-that/gimme.sh | bash
 
 You can also download the [latest release][latest-release] and install it yourself.
 
-# Getting Started
+# Prerequisites
+
+- Kubernetes cluster. Take a look at the [docs][k8s-setup] for instructions on how to do it.
 
-You'll need to have a running kubernetes cluster and configured `kubectl` to talk to it. Take a look at the [docs][k8s-setup] for instructions on how to do it.
+    A couple fast and easy solutions:
 
```

Comment:
Add docker desktop here?

Reply:
Probably worth it once docker for mac k8s is stable (instead of edge).